app/controllers/director/counselors_controller.rb

```diff
@@ -49,6 +49,7 @@ def update
     @counselor = Counselor.find(params[:id])
     @counselor.update(update_counselor_params)
     puts "COUNSELOR UPDATED"
+    redirect_to counselor_path(:id => @counselor.id)
```

Comment:
I believe now they are! They'll redirect to the exact same page when a counselor is created/updated.